storage/src/mem.rs

```diff
@@ -17,9 +17,7 @@
 use crate::NUM_COLS;
 use snarkvm_objects::{DatabaseTransaction, Op, Storage, StorageError};
 
-use parking_lot::RwLock;
```

Comment:
why not just use `parking_lot`? we are going to have it as an indirect dependency regardless, so is there any benefit to using the `std` rendition?

Reply:
I guess. This MemDb seems unused everywhere, as far as I can tell. Will change.